src/core/Header.tsx

```diff
@@ -271,8 +271,9 @@ const Header: React.FC<HeaderProps> = ({
           />
           <div
             id="mobile-menu"
-            className="md:hidden fixed flex flex-col top-[76px] overflow-y-hidden left-0 right-0 mx-12 bg-neutral-000 dark:bg-neutral-1300 rounded-2xl ui-shadow-lg-medium z-20"
+            className="md:hidden fixed flex flex-col top-[76px] overflow-y-hidden mx-12 right-0 w-[calc(100%-24px)] bg-neutral-000 dark:bg-neutral-1300 rounded-2xl ui-shadow-lg-medium z-20"
             style={{
+              maxWidth: "560px",
```

Comment:
We should either do `maxWidth: MAX_MOBILE_MENU_WIDTH` to keep that constant you had going, or go back to `max-w-[560px]` as a class - I think the former as the constant is a good idea